Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -1495,6 +1495,80 @@ def messages() -> Response:
     return Response(status=200)
 
 
+def ring_user():
+    """Rings a user on Teams.
+
+    Notes:
+        This is a ring only! no media plays in case the generated call is answered.
+
+    Returns:
+        None.
+    """
+    bot_id = demisto.params().get('bot_id')
+    integration_context: dict = demisto.getIntegrationContext()
+    tenant_id: str = integration_context.get('tenant_id', '')
+    if not tenant_id:
+        raise ValueError(
+            'Did not receive tenant ID from Microsoft Teams, verify the messaging endpoint is configured correctly.'
+        )
+
+    callback_uri = demisto.args().get('callback_uri', 'https://callback.url')
+
+    # get user to call name and id
+    username_to_call = demisto.args().get('username')
+    users: list = get_users()
+    user_id: str = str()
+    found_member: bool = False
+    for user in users:
+        if username_to_call in {user.get('displayName', ''), user.get('mail'), user.get('userPrincipalName')}:
+            found_member = True
+            user_id = user.get('id', '')
+            break
+    if not found_member:
```

Comment:
you dont need that flag, just check `if user_id:`

Reply:
done